woodpecker-go/woodpecker/client.go

```diff
@@ -120,7 +129,10 @@ func (c *client) open(rawURL, method string, in any) (io.ReadCloser, error) {
 	}
 	req, err := http.NewRequest(method, uri.String(), nil)
 	if err != nil {
-		return nil, err
+		return nil, &ClientError{
```

Comment:
@anbraten Would you revert it here as well?

Reply:
For a similar reason I would avoid extracting / wrapping existing errors there as well.